src/renderer/modals/VoteTron/fields/VotesField.js

```diff
@@ -284,43 +392,45 @@ const AmountField = ({ account, onChangeVotes, transaction, status, bridgePendin
     [explorerView],
   );
 
-  const onUpdateVote = useThrottledCallback((address: string, count: number) => {
-    const newVotes = votes;
-
-    newVotes[address] = parseInt(Number(count));
-
-    if (count <= 0 || Object.keys(newVotes).length > SR_MAX_VOTES) delete newVotes[address];
+  const handleScroll = useCallback(
+    debounce(() => {
```

Comment:
bad pattern. Either use our in-house useDebounceCallback or find another way. This makes the debounce recreate at each render time